src/test/kotlin/lotto/domain/MoneyTest.kt

```diff
@@ -1,10 +1,13 @@
-package lotto.domain.value
+package lotto.domain
 
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Assertions.assertThatThrownBy
 import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.math.BigInteger
+
+internal class MoneyTest {
 
-class MoneyTest {
     @Test
     fun `음수가 들어올 경우 IllegalArgumentException이 발생해야한다`() {
         assertThatThrownBy { Money(-1.toBigInteger()) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThatThrownBy { Money((-1).toBigInteger()) }
```